src/Apps/Order/Components/BankAccountPicker.tsx

```diff
@@ -80,7 +101,7 @@ export const BankAccountPicker: FC<Props> = props => {
 
   return (
     <>
-      {bankAccountsArray.length > 0 && (
+      {bankAccountsArray && bankAccountsArray.length > 0 && (
```

Comment:
is it possible for `me.bankAccounts` (the source of this variable) to be null/undefined? The types above seem to suggest it is always present.

Reply:
`bankAccounts` on `me` can be an array or null. we then extract this `bankAccounts` coming from the API into `bankAccountsArray` which can also include the unsaved bank account. I'll shorten this to` bankAccountsArray?.length` 👍